templates/cinder/config/cinder-volume-config.json

```diff
@@ -12,18 +12,6 @@
       "dest": "/etc/cinder/nfs_shares",
       "owner": "root:cinder",
       "perm": "0640"
-    },
-    {
```

Comment:
@Akrog I see a problem to keep the old ceph mechanism coexisting with extra voumes: in particular if I propagate an extra volume in `/etc/ceph`, via somethin like the following:

```
 extraMounts:
      extraVol:
        - propagation:
          - CinderVolume
          extraVolType: Ceph
          volumes:
          - name: ceph
            projected:
              sources:
              - secret:
                  name: ceph-client-conf
          mounts:
          - name: ceph
            mountPath: "/etc/ceph"
            readOnly: false
```

It won't take effect because when the pod starts (cinder-volume in this specific case), `kolla` overwrite/replace the content of the mounted path (even "merge": true doesn't solve the problem) [1].
Not sure you have a suggestion on how to skip the files copy under certain conditions.

I'm thinking about just removing the old ceph way, and rely entirely on this feature once it merges.
 
 [1] https://docs.openstack.org/kolla/ussuri/admin/kolla_api.html

Reply:
I agree 100%. We should remove the old mechanism.
Fortunately we don't have to worry about backward compatibility at this stage and can do breaking changes  :-)